diff -Nur tslib-1.0/tests/ts_calibrate.c tslib-1.0.orig/tests/ts_calibrate.c
--- tslib-1.0.orig/tests/ts_calibrate.c	2006-08-24 23:02:55.000000000 +0200
+++ tslib-1.0/tests/ts_calibrate.c	2009-09-15 10:47:14.000000000 +0200
@@ -224,9 +224,9 @@
 		for (i = 0; i < 7; i++) printf("%d ", cal.a [i]);
 		printf("\n");
 		if ((calfile = getenv("TSLIB_CALIBFILE")) != NULL) {
-			cal_fd = open (calfile, O_CREAT | O_RDWR);
+			cal_fd = open (calfile, O_CREAT | O_RDWR, 0644);
 		} else {
-			cal_fd = open ("/etc/pointercal", O_CREAT | O_RDWR);
+			cal_fd = open ("/etc/pointercal", O_CREAT | O_RDWR, 0644);
 		}
 		sprintf (cal_buffer,"%d %d %d %d %d %d %d",
 			 cal.a[1], cal.a[2], cal.a[0],
